### ISSUE TITLE
Unraid Template

### ISSUE THREAD

**Issue by e-p-s** [linkwarden/linkwarden#416]
Is your feature request related to a problem? Please describe.
I'm facing difficulties getting the Bookmark app to run on Unraid. Despite my efforts, I have been unable to properly install and use the app along with its database.

Describe the solution you'd like
I would like to see someone create an All-in-One (AIO) template for Unraid that includes the Bookmark app and its associated database. This would simplify the installation and usage on Unraid.

Describe alternatives you've considered
So far, I haven't found an effective solution, and a pre-made Unraid template would be a welcome alternative to streamline the setup.

Additional context
It would be helpful if the community could provide a Unraid template that facilitates the smooth integration of the Bookmark app on Unraid. Any potential adjustments or additional information can be better incorporated this way.

**Comment by Nelinski** [comment]
I agree, would be great to set this up in Unraid via the app store. I've just set this up manually on Unraid as I've already  got postgres running. Really simple following the Linkwarden documentation.

**Comment by e-p-s** [comment]
Nice. Someone did it.

Its not complete AIO, cause you have to set up postgres selfly. But hey... For me, its works. Thanks.

![image](https://github.com/linkwarden/linkwarden/assets/42796829/32a55653-ac62-4e3d-80ce-23c8ad6210a0)
